packages/lesswrong/components/vulcan-forms/Form.tsx

```diff
@@ -713,23 +713,18 @@ export class Form<T extends DbObject> extends Component<SmartFormProps,FormState
 
   // check for route change, prevent form content loss
   checkRouteChange = () => {
+    // TODO FIXME: This was a casualty of upgrading react router v5->v6
+    /*
```

Comment:
It seems like this never did anything anyways, since `getWarnUnsavedChanges` just returns `false`?  Possible I'm missing something though.

Reply:
Yep - that's the case at the moment. It looks like it was actually used back when ForumMagnum was still on react-router 3 though (however long ago that was).